### ISSUE TITLE
Menu is broken

### ISSUE THREAD

**Issue by nagbg** [tailwindlabs/headlessui#267]
In the latent version of vue the components are broken. Even on the demo page the menu doesn’t open on click. In my own build it opens but active state on hover or keyboard navigation does not work.

**Comment by nagbg** [comment]
Hi @RobinMalfait!

Found out it's breaking after Vue 3.0.5.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

Very odd that this starts happening randomly after a some patch updates 🤔 
I'm going to investigate this and report back!

**Comment by RobinMalfait** [comment]
This should be fixed, and will be available in the next release.
You can already try it using `npm install @headlessui/vue@dev` or `yarn add @headlessui/vue@dev`.

Relevant PR: #279
